docs/newbs.md

```diff
@@ -6,13 +6,17 @@ Not sure if your keyboard can run QMK? If it's a mechanical keyboard you built y
 
 ## Overview
 
-There are 5 main sections to this guide:
+There are 6 main sections to this guide:
 
 * [Getting Started](newbs_getting_started.md)
 * [Building Your First Firmware](newbs_building_firmware.md)
 * [Flashing Firmware](newbs_flashing.md)
 * [Testing and Debugging](newbs_testing_debugging.md)
-* [Best Practices](newbs_best_practices.md)
+* [Git Best Practices](newbs_best_practices.md)
 * [Learn More with these Resources](newbs_learn_more_resources.md)
 
 This guide is focused on helping someone who has never compiled software before. It makes choices and recommendations based on that viewpoint. There are alternative methods for many of these procedures, and we support most of those alternatives. If you have any doubt about how to accomplish a task you can [ask us for guidance](getting_started_getting_help.md).
+
+## Additional Resources
+
+* [Thomas Baart's QMK Basics Blog](https://thomasbaart.nl/category/mechanical-keyboards/firmware/qmk/qmk-basics/) - A user created blog covering the basics of how to use QMK Firmware, as seen from a new user.
```

Comment:
Because I'm that guy:

```suggestion
* [Thomas Baart's QMK Basics Blog](https://thomasbaart.nl/category/mechanical-keyboards/firmware/qmk/qmk-basics/) - A user-created blog covering the basics of how to use QMK Firmware, as seen from a new user.
```

Everything else is :+1: IMO.